tests/MyGame/Example/Monster.java

```diff
@@ -344,5 +344,364 @@ public static Monster __lookup_by_key(Monster obj, int vectorLocation, String ke
     public Monster getByKey(String key) {  return __lookup_by_key(null, __vector(), key, bb); }
     public Monster getByKey(Monster obj, String key) {  return __lookup_by_key(obj, __vector(), key, bb); }
   }
+  public MonsterT unpack() {
+    MonsterT _o = new MonsterT();
+    unpackTo(_o);
+    return _o;
+  }
+  public void unpackTo(MonsterT _o) {
+    MyGame.Example.Vec3T _oPos = pos() != null ? pos().unpack() : null;
```

Comment:
Needs to pass the existing value of `_o.pos` to `unpackTo` to not waste allocating another object. Same with any other subjects for which `unpack` is called below. This method is called `unpackTo` which means it should overwrite existing objects if any, not allocate new.

Reply:
OK so I will replace it with pos().unpackTo(_o.getPos())